yt_dlp/networking/_websockets.py

```diff
@@ -90,10 +92,12 @@ class WebsocketsRH(WebSocketRequestHandler):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        for name in ('websockets.client', 'websockets.server'):
+        self.__logging_handlers = []
+        for name in WEBSOCKETS_LOGGERS:
             logger = logging.getLogger(name)
             handler = logging.StreamHandler(stream=sys.stdout)
             handler.setFormatter(logging.Formatter(f'{self.RH_NAME}: %(message)s'))
+            self.__logging_handlers.append(handler)
```

Comment:
This being a list seems harder than it needs to be. We can do this instead:
```suggestion
        self.__logging_handlers = {}
        for name in WEBSOCKETS_LOGGERS:
            logger = logging.getLogger(name)
            handler = logging.StreamHandler(stream=sys.stdout)
            handler.setFormatter(logging.Formatter(f'{self.RH_NAME}: %(message)s'))
            self.__logging_handlers[name] = handler
```
and then later
```py
        for name, handler in self.__logging_handlers.items():
            logging.getLogger(name).removeHandler(handler)
```